ai-data/generative-apis/quickstart.mdx

```diff
@@ -0,0 +1,95 @@
+---
+meta:
+  title: Generative APIs - Quickstart
+  description: Get started with Scaleway Generative APIs for powerful AI-driven content generation. Follow this guide to set up, configure, and make your first API request.
+content:
+  h1: Generative APIs - Quickstart
+  paragraph: Get started with Scaleway Generative APIs for powerful AI-driven content generation. Follow this guide to set up, configure, and make your first API request.
+tags: generative-apis ai-data quickstart
+dates:
+  validation: 2024-08-27
+categories:
+  - ai-data
+---
+
+Generative APIs offer seamless access to pre-configured, serverless endpoints of leading AI models. 
+No need to configure hardware or deploy your own models.
+
+Hosted in European data centers and priced competitively per million tokens used, these APIs enable efficient and scalable integration of AI capabilities into your applications.
+
+
+<Message type="important">
+  This service is free while in beta. [Specific terms and conditions](https://www.scaleway.com/en/contracts/) apply.
+</Message>
+
+<Macro id="requirements" />
+  - A Scaleway account logged into the [console](https://console.scaleway.com)
+  - [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+  - A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+  - Python 3.7+ installed on your system.
+
+## Start with the Generative APIs Playground
+
+Scaleway's Playground is in development, stay tuned!
+
+## Install the OpenAI Python SDK
+
+To start using Generative APIs in your code, you can install the OpenAI Python SDK. Run the following command:
+
+```sh
+pip install openai
+```
+
+<Message type="tip">
+  Ensure Python is installed on your local machine. If you need to install Python, download it from [python.org](https://www.python.org/downloads/).
+</Message>
+
+## Configure and set your API key and service URL
+
+1. Have your [API key](/identity-and-access-management/iam/how-to/create-api-keys/) ready or generate a new one from the Scaleway console.
+2. Use the following code in your Python script:
+
+```py
+from openai import OpenAI
+
+client = OpenAI(
+    base_url="https://api.scaleway.ai/v1",
+    api_key="<SCW_API_KEY>"
+)
+```
+
+Replace `<SCW_API_KEY>` with the API key obtained from your Scaleway account.
+
+<Message type="tip">
+  It is recommended to store your API keys securely using environment variables or [secret management tools](/docs/identity-and-access-management/secret-manager/) to prevent unauthorized access.
```

Comment:
```suggestion
  It is recommended to store your API keys securely using environment variables or [secret management tools](/identity-and-access-management/secret-manager/) to prevent unauthorized access.
```